aiocache/base.py

```diff
@@ -109,18 +110,26 @@ class BaseCache:
     NAME: str
 
     def __init__(
-        self, serializer=None, plugins=None, namespace=None, key_builder=None, timeout=5, ttl=None
+        self,
+        serializer: Optional[BaseSerializer] = None,
+        plugins: Optional[List[Any]] = None,  # aiocache.plugins depends on aiocache.base
+        namespace: str = "",
```

Comment:
This breaks `build_key()` if `None` is passed as an argument here. 

Either we allow `None` here and convert it to `""` inside `__init__()`, or we should add a test that expects to fail when `None` is passed here.

Which would you recommend?

Reply:
A runtime check? Sure, that's fine.

I'd vote for not allowing `None` though and keeping the code simpler.

Reply:
For now, have omitted a runtime check.  Passing `namespace=None` does not cause an error; instead it prepends the "None" namespace to the key.

This is probably unexpected behavior, but doesn't cause any real problem.  I could add a runtime warning to alert the user that `""` (or no argument) is the expected value for a cache constructor.